kernelci.org/docker-compose.yaml

```diff
@@ -1,19 +1,32 @@
 # SPDX-License-Identifier: LGPL-2.1-or-later
 #
-# Copyright (C) 2022 Collabora Limited
-# Author: Guillaume Tucker <guillaume.tucker@collabora.com>
```

Comment:
Why remove this part?

In principle it should be updated to this:
```
# Copyright (C) 2022-2024 Collabora Limited
# Author: Guillaume Tucker <guillaume.tucker@collabora.com>
# Author: Denys Fedoryshchenko <denys.f@collabora.com>
```

Or alternatively, I believe this is also valid:

```
# Copyright (C) 2022-2023 Collabora Limited
# Author: Guillaume Tucker <guillaume.tucker@collabora.com>

# Copyright (C) 2024 Collabora Limited
# Author: Denys Fedoryshchenko <denys.f@collabora.com>
```

In any case, Collabora's copyright shouldn't be removed for 2022 and 2023.

Reply:
It was totally new file rewritten from scratch (well base taken from https://github.com/jakejarvis/hugo-docker ), but nvm, i will put back copyrights.

Reply:
Ah OK, might be a bit of a Git diff artifact then.  Still, about half of the lines aren't changing.  Yeah I think it's safer to keep the copyright from previous years in any case as best practice.